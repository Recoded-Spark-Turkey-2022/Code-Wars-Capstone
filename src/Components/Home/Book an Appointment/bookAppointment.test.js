// import renderer from 'react-test-renderer';
// import BookAppointment from './bookAppointment';


// it('BookAppointment renders correctly', () => {
//   const tree = renderer
//     .create(<BookAppointment/>)
//     .toJSON();
//   expect(tree).toMatchSnapshot();
// });







import React from "react";
import renderer from "react-test-renderer";
import BookAppointment from './bookAppointment';
import { MemoryRouter } from "react-router-dom";

it("renders correctly", () => {
    const test = renderer
        .create(
            <MemoryRouter>
                <BookAppointment/>
            </MemoryRouter>
        )
        .toJSON();
    expect(test).toMatchSnapshot();
});