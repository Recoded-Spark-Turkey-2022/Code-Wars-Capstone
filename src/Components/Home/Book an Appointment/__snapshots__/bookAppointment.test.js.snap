// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="relative h-screen "
  style={
    Object {
      "fontFamily": "Poppins, sans-serif",
    }
  }
>
  <p>
     
  </p>
  <img
    alt="BackGround"
    className="absolute top-0 left-0 z-0 bg-cover   "
    src="BackGround.png"
  />
  <div
    className="z-10 w-6/12 translate-y-1/3  translate-x-2/3 "
  >
    <img
      alt="iltration"
      className="imgsofa"
      src="Towperson.svg"
    />
  </div>
  <div
    className=" absolute top-28  left-40"
  >
    <p
      className="paragraph"
    >
      We Are Here To
    </p>
    <p
      className="paragrapHelp"
    >
      Help
    </p>
    <a
      className="text-xl md:text-lg  lg:px-6 rounded-md box-border p-2 transition-all duration-250 bg-cyan-400 hover:bg-cyan-500 hover:text-white "
      href="#Loge In"
    >
      BOOK AN APPOINTMENT
    </a>
  </div>
</div>
`;
